packages/fxa-settings/src/components/Settings/PageRecentActivity/SecurityEvent.tsx

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+
+enum SecurityEventName {
+  Create = 'account.create',
+  Disable = 'account.disable',
+  Enable = 'account.enable',
+  Login = 'account.login',
+  Reset = 'account.reset',
+  ClearBounces = 'emails.clearBounces',
+}
+
+const getSecurityEventNameL10n = (name: string) => {
```

Comment:
Not ideal, but this gives the most flexibilty in changing texts (ie adding `-v2`) to ids.